README.md

```diff
@@ -22,30 +22,26 @@ The public endpoint for the Valorem Trade API is `https://trade.valorem.xyz`.
 There are two principal user roles in the Valorem Trade API:
 
 - **Maker**: Makers sign offers in response to requests for quotes (RFQs).
-  They are responsible for having the requisite assets when a taker optionally 
+  They are responsible for having the requisite assets when a taker optionally
   fills their signed offer. Makers are presently required to request access to
-  the maker API via the [Valorem discord](https://discord.gg/valorem).
+  the maker API via the [Valorem discord](https://discord.gg/valorem); set to be lifted pending the battle testing of our offer validation protocol.
```

Comment:
Would remove. Let's not make forward looking promises here, but rather speak about the present.